actions/copy.js

```diff
@@ -18,7 +19,7 @@ exports.copy = function(from, to, options) {
   to = path.resolve(to);
   options = options || {};
 
-  if (!glob.hasMagic(from)) {
+  if (!Array.isArray(from) && !glob.hasMagic(from)) {
```

Comment:
We might as well get ride of this `if` all together. I think a large part of this check was to distinguish a glob patter and a static path, which are both string. Now that `from` is overloaded with many forms, the performance gain we get from short-circuiting may no longer out weight the readability of the logic without it. What do you think?